doc/MONSTERS.md

```diff
@@ -315,6 +316,18 @@ Amount of cutting damage added to die roll on monster melee attack.
 
 Monster protection from bashing, cutting, stabbing, acid and fire damage.
 
+## "weakpoints"
+(array of objects, optional)
+
+Weakpoints in the monster's protection.
+
+| field               | description
+| ---                 | ---
+| `name`              | Name of the weakpoint.
+| `coverage`          | Base probability of hitting the weakpoint. May be increased by skill level.
+| `armor_multiplier`  | multipler on the monster's base protection when hitting the weakpoint.
+| `armor_offset`      | offset on the monster's protection, after applying the multiplier.
```

Comment:
I believe this is added to the armour after the multiplier? If so we should make that more clear, like:
```suggestion
| `armor_offset`      | a flat value added to the monster's protection, after applying the multiplier.
```
We could consider a more self explanatory name for the variable, although the only ones I can think of aren't much better.

Reply:
Two thoughts off the top of my head:
- `armor_pen`: This would be similar to the armor pen values described for bullets
- `armor_flat`: This would highlight the fact that it's a flat bonus
Do you have a preference for either?

Reply:
Hm. Is the value subtracted from or added to remaining armour? If it's subtracted, `armor_penalty` would work. If it's added, `armor_modifier` might be the best term, even though it looks a lot like `multiplier`.

Reply:
I renamed `armor_offset` to `armor_penalty`.

Previously, the value was added to remaining armor, but I realized that defaulting it to subtraction would make more sense. Positive modifiers are still support through negative `armor_penalty` values.